src/main/java/appeng/tile/storage/TileDrive.java

```diff
@@ -410,10 +397,20 @@ public void saveChanges( final IMEInventory cellInventory )
 		this.world.markChunkDirty( this.pos, this );
 	}
 
-	@Override
-	public boolean isEmpty()
+	private class CellValidInventoryFilter implements IAEItemFilter
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "CellValidInventoryFilter" class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3985)